src/main/java/org/helpberkeley/memberdata/WorkflowParser.java

```diff
@@ -221,6 +265,13 @@ private boolean isDriverRow(WorkflowBean bean) {
         return Boolean.parseBoolean(driverValue) && (! Boolean.parseBoolean(consumerValue));
     }
 
+    private boolean isMemberRow(WorkflowBean bean) {
+        String consumerValue = bean.getConsumer();
+        String driverValue = bean.getDriver();
+
+        return Boolean.parseBoolean(driverValue) != Boolean.parseBoolean(consumerValue);
```

Comment:
From the caller we should also be auditing that both are not TRUE.  Otherwise we won't catch this until they submit a request to generate driver messages (checked in WorkflowParser.drivers()), and if they just update their request,  we may have missed updated driver changes.